pvlib/tracking.py

```diff
@@ -43,16 +43,26 @@ class SingleAxisTracker(PVSystem):
         between the tracking axes has a gcr of 2/6=0.333. If gcr is not
         provided, a gcr of 2/7 is default. gcr must be <=1.
 
+    side_slope : float, default 0.0
+        The slope of the "system plane" perpendicular to the tracker axes. The
+        "system plane" is defined as the plane that contains all of the tracker
+        axes. EG north-south trackers on a 3-degree eastern slope would have a
+        3-degree side slope, depending on the tracker axis azimuth. Use
```

Comment:
```suggestion
        "system plane" is defined as the plane that contains all of the tracker
        axes. For example, north-south trackers on an east-facing, three-degree
        upward slope would have a 'side_slope' value of 3.0. Use
```
Does the sign of `side_slope` matter?